src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -399,7 +399,15 @@ private void UpdateHref(List<ManifestItemWithContext> manifest, IDocumentBuildCo
 
                 var path = Path.Combine(file.BaseDir, file.File);
                 metadata = ApplyFileMetadata(path, metadata, fileMetadata);
-                return processor.Load(file, metadata);
+                try
+                {
+                    return processor.Load(file, metadata);
+                }
+                catch (Exception)
+                {
+                    Logger.LogError($"Unable to load file: {file.File} via processor: {processor.Name}.");
```

Comment:
re-throw the exception.

---
In reply to: [73640655](https://github.com/dotnet/docfx/pull/573#discussion_r73640655) [](ancestors = 73640655)